templates/inspec/plural_resource.erb

```diff
@@ -16,7 +16,7 @@
 
 <%= lines(autogen_notice :ruby) -%>
 require 'gcp_backend'
-class <%= object.name -%>s < GcpResourceBase
+class <%= object.__product.name.camelize(:upper) -%><%= object.name -%>s < GcpResourceBase
```

Comment:
Modified this to namespace resources so that we can differentiate between things like `redis_instance` and `filestore_instance`.

There is no functional change as nothing references these by name

Reply:
Sounds good! Thanks for telling me that.